docs/guides/building-forms.md

```diff
@@ -393,8 +387,8 @@ That's about it. After reading this guide, here's what you'll learn:
 - The basic HTML `form` element
 - Understanding forms with React.js
 - Validating forms data with and without JavaScript
-- Using Next.js Sereverless Functions to handle `req` and `res` from the client and server
+- Using Next.js to handle `req` and `res` from the client and server
 
-Next.js provide you with an all-in solution for hosting web forms and handling submissions with serverless functions. Use Next.js as the client for your JavaScript-based interaction and Vercel for its powerful serverless functions to handle the server.
+Next.js provides you with an all-in solution for hosting web forms and handling submissions. Self-hosted, Next.js API routes can run on your Node.js server, deployed to Vercel.
```

Comment:
I think this could be made clearer too. You can self host your next js app, or you can deploy it to a cloud provider, such as Vercel.